ansible/roles/packages/tasks/repo-debian.yaml

```diff
@@ -1,8 +1,10 @@
 ---
+  - name: clean and update apt cache
+    shell: apt-get clean && apt-get update
```

Comment:
This looks like a different commit https://github.com/mesosphere/konvoy-image-builder/pull/207?

Reply:
I rebased it off #207. I will rebase this PR off main branch once #207 is merged. That should remove this commit.